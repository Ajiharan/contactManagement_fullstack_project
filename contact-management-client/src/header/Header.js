import React from "react";
import styled from "styled-components";
const Header = () => {
  console.log("header");
  return (
    <NavbarTop>
      <div className="navbarTop__container">
        <h2 className="text-center">Contact Management System</h2>
      </div>
    </NavbarTop>
  );
};

const NavbarTop = styled.div`
  padding: 10px;
  min-height: 50px;
  background-repeat: no-repeat;
  border-bottom: 2px solid rgb(245, 187, 80);
  background-size: cover;
  background-position: bottom;
  box-shadow: 0px 4px 4px rgba(255, 255, 255, 0.25),
    0px 2px 4px rgba(225, 233, 238, 0.2);
  .navbarTop__container {
    display: flex;
    align-items: center;
    justify-content: center;
    flex-direction: column;
    h2 {
      background: linear-gradient(
        to left,
        rgb(245, 161, 82),
        rgb(223, 173, 37)
      );
      -webkit-background-clip: text;
      display: inline;
      -webkit-text-fill-color: transparent;
      font-size: 3rem;
    }
    .navbarTop__containerImages {
      display: flex;
      img {
        background-color: rgb(253, 252, 251);
        margin: 10px;
        padding: 4px;
        border-radius: 100%;
        box-shadow: 0px 10px 10px rgba(206, 148, 60, 0.774),
          0px 20px 20px rgba(240, 187, 74, 0.15),
          0px 30px 30px rgba(240, 187, 74, 0.1);
        height: 50px;
        object-fit: contain;
      }
    }
  }
`;

export default Header;
