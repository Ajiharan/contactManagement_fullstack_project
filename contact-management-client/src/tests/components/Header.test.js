import React from "react";
import { shallow } from "enzyme";
import Header from "../../header/Header";

test("should test header component", () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
});
