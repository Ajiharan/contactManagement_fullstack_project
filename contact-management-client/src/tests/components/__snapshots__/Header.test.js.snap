// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test header component 1`] = `
<styled.div>
  <div
    className="navbarTop__container"
  >
    <h2
      className="text-center"
    >
      Contact Management System
    </h2>
  </div>
</styled.div>
`;
